awscli/customizations/history/db.py

```diff
@@ -83,11 +79,53 @@ def _encode_mutable_mapping(self, obj):
     def _encode_datetime(self, obj):
         return obj.isoformat()
 
+    def _try_encode(self, obj):
+        try:
+            obj = obj.decode('utf-8')
+        except UnicodeDecodeError:
+            obj = '<Byte sequence>'
+        return obj
+
+    def _remove_non_unicode_stings(self, obj):
+        if isinstance(obj, str):
```

Comment:
What's your thoughts on using `binary_type` here instead?

Reply:
I match exactly what the JSON encoder does so I would be hesitant to deviate.